setup.py

```diff
@@ -40,7 +40,7 @@
 if __name__ == '__main__':
     setup(
         name='pyensembl',
-        version="0.6.11",
+        version="0.7.0",
```

Comment:
Seems good, for pickling and other changes (since the Genome API might be a surprising difference as well)